src/ui/components/filters/filteroptionscomponent.js

```diff
@@ -487,19 +507,38 @@ export default class FilterOptionsComponent extends Component {
     return (option && filter) ? option.indexOf(filter) : -1;
   }
 
+  /**
+   * Clears all selected options. If this component is not part of a FilterBox,
+   * also send the REMOVED_FILTER analytics event. Otherwise, let FilterBox do it.
+   */
   clearOptions () {
+    if (!this.config.isDynamic) {
```

Comment:
Are we relying on config.isDynamic to indicate if this came from FilterBox? If so, that's a bit of a code smell. That kind of logic can be dangerous as isDynamic was not meant to represent whether or not FilterBox was the parent. If FilterBox is not being used with Facets, but with a collection of normal FilterOptions, will we see duplicate analytics events getting fired?

Reply:
Yup there'l be dupes, clearOptions is called both when FilterBox resets all of its FilterOptions as well as when FilterOptions resets itself

Reply:
removing analytics event from this PR so this is no longer an issue, otherwise would have added a new 'isChildOfFilterBox' config like we discussed